### PR TITLE
Ridley Leisy - Unit 2 - Regression 2

### PR DESCRIPTION
**Pull request by RidleyLeisy**

### CODE REVIEW

**Comment by axrd** [comment]
# Module 01 - Log-Linear Regression, Feature Engineering

**Overall**: *Really* well done! You crushed a really tricky assignment! Also, one of the cleanest notebooks I've seen all day. 

**Feedback**: That is the perfect amount of commenting 👌 Keep it up!

## Load in the Data
- Glob is super useful, huh? 👍
- Great thinking splitting them all into different dataframes 👍  💯 

## Select Features 
- Proper merging and you accounted for renaming Cols 👍 

## Wrangling your Data
- Account for skew 👍 
- Handle temporal values 👍 
- Great job making your code functional! 👍 👏 💯 

## Train/Val/Split
- Proper implementation, using train_test_split twice to create a validation set 👌 

## Fit a Model
- Yup. Nicely done. 

## Validation RMSLE (or RMSE)
- Darn good RMSE! Curious how some boosting will impact it.

**Comment by axrd** [comment]
# Module 02 - Gradient Boosting, More Feature Engineering

*Overall*: Nicely done! Were you able to figure out a way around Jupyter crashing on you?
*Feedback*: More commenting than that. I know it's annoying and repetitive but please implement it!
## More Feature Engineering
- Seeing some nice additions. What else can you engineer? Refer to today's lecture for ideas!

## Implement Gradient Boosting
- There's a weird `NameError` at the end of your NB where `'numerics' is not defined`. Were you able to figure that out?

### Reference:
[This is the best explanation of Gradient Boosting I've found so far](https://www.reddit.com/r/explainlikeimfive/comments/86dx3u/eli5_what_is_gradient_boosting_and_how_can_it_be/), in case you want to check it out.

Also, take a look at [Ryan's lecture notebook](https://colab.research.google.com/drive/1LAGRZ5jjO-35hHHTCOtO2VvxwRdwBy-o#scrollTo=zuzgmaBfYiiI) from today for a great reference of a possible solution.
